generic_command.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+
+import iterm2
+
+
+SCRIPT = 'SCRIPT'
+PREFIX = 'PREFIX'
+
+async def main(connection):
+    component = iterm2.StatusBarComponent(
+        short_description='Execute a user defined command',
+        detailed_description='A super simple component where both the command creating the text and a prefix can be defined by the user',
+        exemplar='⎈ docker-for-desktop',
+        update_cadence=2,
```

Comment:
This is hard-coded to 2 seconds, which might be too slow or unnecessarily frequent for some users. I'm not sure if this can be set dynamically by a knob?

Reply:
yeah, two seconds is pretty frequent, I'll change that to 7 for the general case. Why 7, it sounds good. I don't think it is configurable as a knob.